README.md

```diff
@@ -1,16 +1,33 @@
-# Lane Keeping Assist via CNN
+# Lane Keeping Assist
+Welcome!
 
-### Repository Structure
+## :classical_building: Repository Structure
 ```bash
 ├── README.md       
 ├── subject.pdf   # subject of the project
 │ 
 ├── src/          # .py .ipynb files for building a model and control a car
 │   ├── main.py
```

Comment:
please add new line after this

Reply:
✅

Reply:
new line is not same as previous one. please check the color

Reply:
updated